test/Jamfile.v2

```diff
@@ -20,6 +20,7 @@ run test_rapidxml.cpp ;
 run test_info_parser.cpp ;
 run test_json_parser.cpp ;
 run test_json_parser2.cpp ;
+run test_json_parser3.cpp ;
```

Comment:
This should probably be `compile` instead of `run`, as we aren't testing anything besides compilation.